resources/auth_pubkey.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Yes, I guess it's a dummy, should actually correspond to something in heimdall or be removed altogether